status/flowcells.py

```diff
@@ -201,7 +201,13 @@ class FlowcellNotesDataHandler(SafeHandler):
     def get(self, flowcell):
         self.set_header("Content-type", "application/json")
         try:
-            p = lims.get_containers(name=flowcell[8:])[0]
+            if flowcell[7:].startswith('00000000'):
+                #Miseq
```

Comment:
@senthil10 I remember that once you were able to tell if a FC was highRun, Rapid, miSeq/HiSeq by looking at some part of the flowcell. Can you share with us your wisdom, maybe there is a better way to distinguish MiSeq form HiSeq...